packages/devtools_app/lib/src/inspector/diagnostics_node.dart

```diff
@@ -11,40 +11,22 @@ import 'package:vm_service/vm_service.dart';
 import '../ui/fake_flutter/fake_flutter.dart';
 import '../ui/icons.dart';
 import '../utils.dart';
+import 'enum_deserializer.dart';
 import 'flutter_widget.dart';
 import 'inspector_service.dart';
 
 Map<K, V> _invertMap<K, V>(Map<V, K> inverted) => Map.fromEntries(
     inverted.entries.map((entry) => MapEntry(entry.value, entry.key)));
 
-const Map<String, DiagnosticLevel> nameToDiagnosticLevel = {
-  'hidden': DiagnosticLevel.hidden,
-  'fine': DiagnosticLevel.fine,
-  'debug': DiagnosticLevel.debug,
-  'info': DiagnosticLevel.info,
-  'warning': DiagnosticLevel.warning,
-  'hint': DiagnosticLevel.hint,
-  'summary': DiagnosticLevel.summary,
-  'error': DiagnosticLevel.error,
-  'off': DiagnosticLevel.off,
-};
+final Map<String, DiagnosticLevel> nameToDiagnosticLevel =
```

Comment:
change the type of this field to just be the EnumSerializer and cleanup the uses.

We can make breaking changes to package:devtools_app without causing problems.

Reply:
initially I was trying to do that but I saw that this Map is being used in `_invertMap()` for `diagnosticLevelToName` and `treeStyleToName`, any suggestion on how to make the changes?
Should we extend the class to change the method name from `deserialize` to `lookup` and  include new method called `reverseLookup()`?

Reply:
or `deserialize` and `serialize` ? 
So the class name would be `EnumSerializationUtils` ?

Reply:
decided to go with `EnumUtils` class name and `getName` & `getEnum` methods for now, any suggestions?